CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Liana daemon and GUI release notes
 
+## 7.0
+
+This release introduces support for Wizardsardine servers as an optional backend for the Liana GUI
```

Comment:
Do we call it Wizardsardine servers now? I thought it was Liana Connect? (also present below)